packages/nuxt/src/components/module.ts

```diff
@@ -175,7 +176,10 @@ export default defineNuxtModule<ComponentsOptions>({
       for (const component of newComponents) {
         if (!(component as any /* untyped internal property */)._scanned && !(component.filePath in nuxt.vfs) && isAbsolute(component.filePath) && !existsSync(component.filePath)) {
           // attempt to resolve component path
-          component.filePath = await resolvePath(component.filePath, { fallbackToOriginal: true })
+          const resolvedPath = resolveModulePath(resolveAlias(component.filePath), { try: true, extensions: nuxt.options.extensions })
+          if (resolvedPath) {
+            component.filePath = normalize(resolvedPath)
```

Comment:
it is so common (and easy to forget!), i'm thinking if exsolve should have tiny built-in slash normalizer.